wsgiservice_restplus/namespace.py

```diff
@@ -81,6 +82,8 @@ def wrapper(cls):
                 if cls._path != url:
                     raise ValueError
 
+            cls.internal = kwargs.get('internal', False)
```

Comment:
This is dangerous and we should avoid this. Instead we should perform this in `add_resource`.

The problem is that we are adding a property to the class. We could potentially overwrite an existing property called "internal"